features/features.go

```diff
@@ -498,6 +498,15 @@ var (
 				enableForClusterProfile(Hypershift, configv1.Default, configv1.TechPreviewNoUpgrade).
 				mustRegister()
 
+	FeatureGateExternalOIDCWithAdditionalClaimMappings = newFeatureGate("ExternalOIDCWithUIDAndExtraClaimMappings").
+				reportProblemsToJiraComponent("authentication").
+				contactPerson("bpalmer").
+				productScope(ocpSpecific).
+				enhancementPR("https://github.com/openshift/enhancements/pull/1777").
+				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+				enableForClusterProfile(Hypershift, configv1.TechPreviewNoUpgrade).
```

Comment:
Does this also need `DevPreviewNoUpgrade`?

Reply:
It doesn't need it but it doesn't hurt. I'll add it.

Reply:
Should be included in the latest push.